vcluster/deploy/topologies/isolated-workloads.mdx

```diff
@@ -6,16 +6,34 @@ sidebar_position: 2
-1. vCluster enforces a [Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/) on syncer level, which means that for example pods that try to run as a privileged container or mount a host path will not be synced to the host cluster. Current valid options are either baseline (default in isolated mode) or restricted. This works for every Kubernetes version regardless of Pod Security Standard support, as this is implemented in vCluster directly. Rejected pods will stay pending in the vCluster and in newer Kubernetes version they will be denied by the admission controller as well.
-2. vCluster deploys a [resource quota](https://kubernetes.io/docs/concepts/policy/resource-quotas/) as well as a [limit range](https://kubernetes.io/docs/concepts/policy/limit-range/) alongside the vCluster itself. This allows restricting resource consumption of vCluster workloads. If enabled, sane defaults for those 2 resources are chosen.
-3. vCluster deploys a [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/) alongside itself that will restrict access of vCluster workloads as well as the vCluster control plane to other pods in the host cluster. (only works if your host [cluster CNI supports network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/#prerequisites))
+This feature imposes a couple of restrictions on vCluster workloads to make sure
+they do not break out of their virtual environment:
+
+1. vCluster enforces a [Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/)
+on syncer level, which means that for example pods that try to run as a privileged
+container or mount a host path will not be synced to the host cluster. Current valid
+options are either baseline (default in isolated mode) or restricted. This works
+for every Kubernetes version regardless of Pod Security Standard support, as this
+is implemented in vCluster directly. Rejected pods will stay pending in the vCluster
+and in newer Kubernetes version they will be denied by the admission controller
+as well.
+2. vCluster deploys a [resource quota](https://kubernetes.io/docs/concepts/policy/resource-quotas/)
+as well as a [limit range](https://kubernetes.io/docs/concepts/policy/limit-range/)
+alongside the vCluster itself. This allows restricting resource consumption of
+vCluster workloads. If enabled, sane defaults for those 2 resources are chosen.
+3. vCluster deploys a [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+alongside itself that will restrict access of vCluster workloads as well as the
+vCluster control plane to other pods in the host cluster. (only works if your host
+[cluster CNI supports network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/#prerequisites))
+
+You can adjust isolation settings through helm values (vcluster.yaml). To configure
```

Comment:
We want to move away from referencing and calling `helm values` even if that is technically what it is and call it only `vcluster.yaml. 

I'd update this to:

Adjust your isolation settings through the `vcluster.yaml` of the vCluster: